sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,197 @@
+WITH standardized_country AS ( -- CTE for all_subscriptions
+  SELECT
+    raw_country AS country,
+    standardized_country AS country_name,
+  FROM
+    mozdata.static.third_party_standardized_country_names
+),
+stripe_subscriptions_history AS (-- CTE for all_subscriptions
+  SELECT
+    *,
+    CONCAT(
+      subscription_id,
+      COALESCE(
+        CONCAT(
+          "-",
+          NULLIF(ROW_NUMBER() OVER (PARTITION BY subscription_id ORDER BY valid_from), 1)
+        ),
+        ""
+      )
+    ) AS subscription_sequence_id
+  FROM
+    `mozdata.subscription_platform.stripe_subscriptions_history`
+  WHERE
+    -- Only include the current history records and the last history records for previous plans.
+    (valid_to IS NULL OR plan_ended_at IS NOT NULL)
+    AND status NOT IN ("incomplete", "incomplete_expired")
+),
+stripe_subscriptions AS (-- CTE for all_subscriptions
+  SELECT
+    -- user_id,
+    customer_id,
+    subscription_sequence_id AS subscription_id,
+    IF(
+      subscription_sequence_id != subscription_id,
+      subscription_id,
+      NULL
+    ) AS original_subscription_id,
+    plan_id,
+    status,
+    synced_at AS event_timestamp,
+    IF(
+      (trial_end > TIMESTAMP(CURRENT_DATE) OR ended_at <= trial_end),
+      NULL,
+      COALESCE(plan_started_at, subscription_start_date)
+    ) AS subscription_start_date,
+    --first subscription start date associated with the subscription id
+    IF(
+      (trial_end > TIMESTAMP(CURRENT_DATE) OR ended_at <= trial_end),
+      NULL,
+      subscription_start_date
+    ) AS original_subscription_start_date,
+    IF(plan_started_at IS NOT NULL, "Plan Change", NULL) AS subscription_start_reason,
+    -- created,
+    trial_start,
+    trial_end,
+    canceled_at,
+    canceled_for_customer_at,
+    cancel_at,
+    cancel_at_period_end,
+    COALESCE(plan_ended_at, IF(ended_at < TIMESTAMP(CURRENT_DATE), ended_at, NULL)) AS ended_at,
+    IF(plan_ended_at IS NOT NULL, "Plan Change", NULL) AS ended_reason,
+    -- fxa_uid,
```

Comment:
I'd recommend including `fxa_uid`, as it will allow us to correlate multiple subscriptions for the same FxA user.

Reply:
done